SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -280,6 +281,69 @@ void USpatialNetDriver::CreateAndInitializeCoreClasses()
 	}
 }
 
+void USpatialNetDriver::CreateServerWorkerEntity(int AttemptCounter)
+{
+	const WorkerRequirementSet WorkerIdPermission{ { FString::Format(TEXT("workerId:{0}"), { Connection->GetWorkerId() }) } };
+
+	WriteAclMap ComponentWriteAcl;
+	ComponentWriteAcl.Add(SpatialConstants::POSITION_COMPONENT_ID, WorkerIdPermission);
+	ComponentWriteAcl.Add(SpatialConstants::METADATA_COMPONENT_ID, WorkerIdPermission);
+	ComponentWriteAcl.Add(SpatialConstants::ENTITY_ACL_COMPONENT_ID, WorkerIdPermission);
+	ComponentWriteAcl.Add(SpatialConstants::INTEREST_COMPONENT_ID, WorkerIdPermission);
+
+	QueryConstraint constraint;
+	constraint.EntityIdConstraint = SpatialConstants::INITIAL_GLOBAL_STATE_MANAGER_ENTITY_ID;
+
+	Query query;
+	query.Constraint = constraint;
+	query.FullSnapshotResult = true;
+
+	ComponentInterest queries;
+	queries.Queries.Add(query);
+
+	Interest interest;
+	interest.ComponentInterestMap.Add(SpatialConstants::POSITION_COMPONENT_ID, queries);
+
+	TArray<Worker_ComponentData> Components;
+	Components.Add(Position().CreatePositionData());
+	Components.Add(Metadata(FString::Format(TEXT("WorkerEntity:{0}"), { Connection->GetWorkerId() })).CreateMetadataData());
+	Components.Add(EntityAcl(WorkerIdPermission, ComponentWriteAcl).CreateEntityAclData());
+	Components.Add(interest.CreateInterestData());
+
+	Worker_RequestId RequestId = Connection->SendCreateEntityRequest(MoveTemp(Components), nullptr);
+
+	CreateEntityDelegate OnCreateWorkerEntityResponse;
+	OnCreateWorkerEntityResponse.BindLambda([this, AttemptCounter](const Worker_CreateEntityResponseOp& Op)
+	{
+		if (Op.status_code == WORKER_STATUS_CODE_SUCCESS)
+		{
+			return;
+		}
+
+		if (Op.status_code != WORKER_STATUS_CODE_TIMEOUT)
```

Comment:
So based on past experience I don't know if we only want to retry on TIMEOUT, things that are timeouts can look like other status codes.

Reply:
Based on the description of status codes this is the only one that made sense to retry on when it comes to entity creation. Which other status codes do you want to retry on?

Reply:
Building against broken status codes sounds worse - @Vatyx is there a ticket you know where erroneous status codes are being tracked?